docs/src/getting-started/cli.md

```diff
@@ -73,15 +74,17 @@ export MIDEN_FAUCET_NODE_URL=https://rpc.testnet.miden.io
 export MIDEN_FAUCET_ACCOUNT_PATH=./faucet.mac
 export MIDEN_FAUCET_NETWORK=testnet
 export MIDEN_FAUCET_EXPLORER_URL=https://testnet.midenscan.com
+export MIDEN_FAUCET_MAX_CLAIMABLE_AMOUNT=1000000000
 
 # Proof of Work
 export MIDEN_FAUCET_POW_SECRET=your-secret-here
 export MIDEN_FAUCET_POW_BASELINE=12
 export MIDEN_FAUCET_POW_CHALLENGE_LIFETIME=30s
-export MIDEN_FAUCET_POW_GROWTH_RATE=1
+export MIDEN_FAUCET_POW_CLEANUP_INTERVAL=2s
+export MIDEN_FAUCET_POW_GROWTH_RATE=0.1
+export MIDEN_FAUCET_POW_BASE_DIFFICULTY_AMOUNT=100000000
```

Comment:
This should be `MIDEN_FAUCET_BASE_AMOUNT`, right? Also, I'd move this setting to the "Basic configuration" section.